openapi/multi-yaml/index.yaml

```diff
@@ -79,6 +79,223 @@ tags:
+          }
+        }
+      }
+      ```
+
+      ##### Example Response: Wallet Payment Method Created
+      ```json
+      {
+        "id": "pm_abc123xyz",
+        "type": "payment_method",
+        "data": {
+          "signature": "3aGWnUznQ",
+          "account_id": "acc_123wallet",
+          "status": "valid",
+          "card": {
+            "id": "pm_abc123xyz",
+            "name": "Lindsay Whalen",
+            "acct_last_four": "4242",
+            "brand": "visa",
+            "token": "pm_abc123xyz",
+            "month": "5",
+            "year": "2025",
+            "metadata": "info",
+            "address_line1_check": "pass",
+            "address_postal_code_check": "pass",
+            "bin_details": {
+              "type": "Credit",
+              "card_brand": "Visa",
+              "card_class": "Consumer",
+              "country": "United States of America",
+              "issuer": "WELLS FARGO BANK, N.A.",
+              "funding_source": "Credit"
+            }
+          }
+        }
+      }
+      ```
+      2. Process payments for any sub-account using the wallet payment method token. The system automatically:
+        - Validates the wallet payment method can be used by the sub-account
+        - Creates a cloned payment method owned by the sub-account
+        - Returns the new sub-account-specific token in the payment response
+
+        ### [Payments API](http://localhost:3001/api-spec#tag/Payments/operation/CreatePayment) Example
+
+        ##### Example Request: Create Payment Using Wallet Token
+        ```json
+        {
+          "amount": 1000,
+          "currency": "usd",
+          "payment_method": {
+            "token": "pm_abc123xyz"
+          }
+        }
+        ```
+
+      ##### Example Response: Payment with Auto Cloned Token
+      ```json
+      {
+        "id": "py_xyz789abc",
+        "type": "payment",
+        "data": {
+          "amount": 1000,
+          "currency": "usd",
+          "status": "succeeded",
+          "payment_method": {
+            "id": "pm_autoclonedxyz789",
+            "signature": "4bHXoVaoR",
+            "account_id": "acc_456subaccount",
+            "status": "valid",
+            "card": {
+              "id": "pm_autoclonedxyz789",
+              "name": "Lindsay Whalen",
+              "acct_last_four": "4242",
+              "brand": "visa",
+              "token": "pm_autoclonedxyz789",
+              "month": "5",
+              "year": "2025"
+            }
+          }
+        }
+      }
+      ```
+
+      ### [Checkout Flow](http://localhost:3001/api-spec#tag/Checkouts/operation/CreateCheckout) Example
+
+      When using platform wallet payment methods with checkouts:
+      1. Create a checkout for the sub-account, specifying a payment method group from the wallet account
+      2. Complete the checkout using a payment method token from the wallet account
+
+      ##### Example Request: Create Checkout
```

Comment:
Same question here: should this include the whole request like we do [here, under Create a checkout](https://docs.justifi.tech/api-spec#tag/Checkout-via-API) or change the headline to `Example request body: ...`
I think I prefer adding the whole request, this way it shows which Sub-Account header to use.